src/lms-ds-loader/edfi_lms_ds_loader/migrator.py

```diff
@@ -19,18 +19,18 @@
 MIGRATION_SCRIPTS = [
     # CAUTION: these scripts will run in order from "top to bottom", so it is
     # critical to maintain the script order at all times.
-    "initialize_lms_database",
-    "create_processed_files_table",
-    "create_user_tables",
-    "create_section_tables",
-    "create_assignment_tables",
-    "create_section_association_tables",
-    "create_assignment_submission_tables",
-    "create_section_activity_tables",
-    "create_system_activity_tables",
-    "create_attendance_tables",
-    "remove_startdate_enddate_from_sectionassociation",
-    "add_mapping_columns_for_edfi_student_and_section",
+    "0001_initialize_lms_database",
+    "0002_create_processed_files_table",
+    "0003_create_user_tables",
+    "0004_create_section_tables",
+    "0005_create_assignment_tables",
+    "0006_create_section_association_tables",
+    "0007_create_assignment_submission_tables",
+    "0008_create_section_activity_tables",
+    "0009_create_system_activity_tables",
+    "0010_create_attendance_tables",
+    "0011_remove_startdate_enddate_from_sectionassociation",
+    "0012_add_mapping_columns_for_edfi_student_and_section",
```

Comment:
We could do it by convention, simply look for the .SQL files in the folder, split by '_' and get the first part, then sort and execute it, so that we don't need to hardcode the migration names. Not saying it should be done on this ticket, it is just an idea.

Reply:
FYI, this is how the code in my end to end integration tests loads these files.